tools/wake/main.cpp

```diff
@@ -168,15 +168,14 @@ void hide_internal_jobs(std::vector<std::vector<std::string>> &out) {
   out.push_back({"tags NOT LIKE '%<d>inspect.visibility=hidden<d>%'", "tags IS NULL"});
 }
 
-void inspect_database(const CommandLineOptions &clo, Database &db, const std::string &wake_cwd) {
-  // tagdag is technically a db inspection, but its very different from the
-  // rest, just handle it and exit.
```

Comment:
Could you copy this comment over as well?  Maybe update it slightly to mention `history` as well, since it is indeed very different from both.